spec/models/post_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) { create(:user) }
+
+  it 'is valid with valid attributes' do
+    post = build(:post, author: user)
+    expect(post).to be_valid
+  end
+
+  it 'is not valid without a title' do
+    post = build(:post, title: nil, author: user)
+    expect(post).not_to be_valid
+  end
+
+  it 'is not valid with a title exceeding 250 characters' do
+    post = build(:post, title: 'a' * 251, author: user)
+    expect(post).not_to be_valid
+  end
+
+  it 'is not valid without text' do
+    post = build(:post, text: nil, author: user)
+    expect(post).not_to be_valid
+  end
+
+  it 'updates the user posts counter after saving' do
+    # post = create(:post, author: user)
+    expect(user.reload.posts_counter).to eq(0)
```

Comment:
Nice job so far, you are trying to test here whether the user post counter is updated after saving, but you haven't actually added a post😉. Kindly add a post and verify that the post_counter equals the number of posts you added👍🏼.